src/Libraries/CoreNodes/FileSystem.cs

```diff
@@ -141,16 +146,18 @@ public static void AppendText(string filePath, string text)
         /// <summary>
         ///     Combines multiple strings into a single file path.
         /// </summary>
-        /// <param name="paths">String to combine into a path.</param>
-        public static string CombinePath(params string[] paths)
+        /// <param name="strings">String to combine into a path</param>
```

Comment:
Please make this `Strings to combine into a path` (pluralize `String`)

Reply:
Addressed in a22e753

Reply:
@mmisol ok the two things left to fix were addressed.